pkg/store/bucket.go

```diff
@@ -114,12 +114,12 @@ func newBucketStoreMetrics(reg *prometheus.Registry, s *BucketStore) *bucketStor
 	var m bucketStoreMetrics
 
 	m.blockDownloads = prometheus.NewCounter(prometheus.CounterOpts{
```

Comment:
misleading variable name then?

Reply:
Yea, I'll redo metrics for `BucketStore` soonish.